### PR TITLE
ps: additional paths now work

### PR DESCRIPTION
**Pull request by rminnich**
Suppose you have a lot of hosts' /proc mounted in /tmp/netproc/*
You would do this
UROOT_PSPATH=/proc:/tmp/netproc/* ps aux | grep dnsmasq

You see:
     PID     PGRP      SID TTY             STAT         TIME  COMMAND
a22/1996     1995     1995 ?               S        00:00:00  dnsmasq
    2069     2068     2068 ?               S        00:00:00  dnsmasq

you might even export UROOT_PSPATH so you always see what you want to see.

The path can have more than 2 elements, of course. The code assumes the first
path element is the local one.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

**Comment by rminnich** [comment]
this gets us a lot of this: http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.100.2633&rep=rep1&type=pdf

but we don't need pid masquerading any more.

If we extend the linux /proc so we can kill and trace by writing into files in /proc, well, we're there.
But for now this is just the bits that give you a simple remote ps
and *how* /proc gets set up is up to you. I will likely try sshfs. 

I've done this before with 9p and ... too slow. 9p issue. SAD!
Anyway I had this itch to scratch, it's scratched, I may use this on the tyan debug, we'll see.

more ps cleanup is coming. There's just so much to do ...

**Comment by rminnich** [comment]
and in case you wonder, yes I did implement that writeable ctl and such in linux /proc ca., IIRC, 2006 with some folks from U. Wisc.. The patches are long lost. The linux community had *0* interest ("we have ptrace, what more to you need?"). yeesh.

File: cmds/ps/ps_test.go [hunk 1]
```diff
@@ -81,7 +90,8 @@ Uid:	110	110	110	110
 			}
 		}
 		c := testutil.Command(t, "aux")
-		c.Env = append(c.Env, "UROOT_PSPATH="+filepath.Join(d, "*/stat"))
+		psp := fmt.Sprintf("UROOT_PSPATH=%s:%s", d, filepath.Join(d, "srv"))
```
Comment (hugelgupf) [inline cmds/ps/ps_test.go]: why an env var and not a flag? flag has the added utility of being listed when you do a -h

Comment (rminnich) [reply cmds/ps/ps_test.go]: the env variable is inherited so, e.g., you can set up an environment for a shell and subshells and the behavior will work, even should you (e.g.) run a ps command from vi or emacs or whatever.

in plan 9 we would do this with union mount over /proc, but this is unix and the major way unix people get the effect of union mounts is environment variables with : separated paths: PATH, GOPATH, and so on.

**Pull request merged**